pilot/pkg/networking/core/cluster_traffic_policy.go

```diff
@@ -348,6 +358,14 @@ func getDefaultCircuitBreakerThresholds() *cluster.CircuitBreakers_Thresholds {
 	}
 }
 
+// getDefaultCircuitBreakerThresholdsRetryBudget returns a copy of the default circuit breaker thresholds retry budget for the given traffic direction.
+func getDefaultCircuitBreakerThresholdsRetryBudget() *cluster.CircuitBreakers_Thresholds_RetryBudget {
+	return &cluster.CircuitBreakers_Thresholds_RetryBudget{
+		BudgetPercent: &xdstype.Percent{Value: math.MaxFloat64},
+		MinRetryConcurrency: &wrapperspb.UInt32Value{Value: math.MaxUint32},
```

Comment:
Why default values here not the same as in the defaults reported in the comments [here](https://github.com/istio/api/pull/3249/files#diff-12899beb695152c851fb77fb9a8983897fe1c9c17eaf7665b146b8d55a9a5c18R663-R673)?
```suggestion
		BudgetPercent: &xdstype.Percent{Value: 20},
		MinRetryConcurrency: &wrapperspb.UInt32Value{Value: 3},
```